cnv_release_notes/cnv_release_notes.adoc

```diff
@@ -11,65 +11,91 @@ include::modules/cnv_introduction_to_cnv.adoc[leveloffset=+2]
-console. https://bugzilla.redhat.com/show_bug.cgi?id=1640505[(BZ#1640505)]
-
-* In the Create VM Wizard, using the `PXE` source option with the `Start VM on 
-creation` option will result in the boot order not changing after stopping 
-and starting the VM. The VM will boot from the PXE NIC instead of the 
-disk. https://bugzilla.redhat.com/show_bug.cgi?id=1648245[(BZ#1648245)] 
-** Restarting the VM will not solve this issue.
-https://bugzilla.redhat.com/show_bug.cgi?id=1647447[(BZ#1647447)]
-** *Workaround:* Stop and start the VM again, and the boot order should change 
-as expected.
+* When installing {ProductName} 1.4, the `ansible-playbook` command fails
+if the `multus` image and its underlying layers are not pulled within the
+timeout period. As a workaround, wait a few minutes and try the command again.
+https://bugzilla.redhat.com/show_bug.cgi?id=1664274[(BZ#1664274)]
 
-* Deploying with `kubevirt-apb` and creating a VM from template may result 
-in the following error: 
-+
-----
-Unable to pull image: Back-off pulling image 
-"registry:5000/kubevirt/fedora-cloud-registry-disk-demo:devel"
-----
-+
-This may cause VM creation to get stuck. 
-https://bugzilla.redhat.com/show_bug.cgi?id=1632255[(BZ#1632255)]
+* The `virtctl image-upload` command fails if the `--uploadproxy-url` value
+ends with a trailing slash. If you use a custom URL, ensure that it does not end
+with a trailing slash before running the command.
+https://bugzilla.redhat.com/show_bug.cgi?id=1660888[(BZ#1660888)]
+
+* The limit for compute node devices is currently 110. This limit cannot be
+configured, but scaling up to more than 110 devices will be supported in a
+future release.
+https://bugzilla.redhat.com/show_bug.cgi?id=1673438[(BZ#1673438)]
+
+* When uploading or importing a disk image to a PVC, the space allocated for the
+PVC must be at least `2 * actual image size + virtual image size`. Otherwise,
+the virtual machine does not boot successfully.
+https://bugzilla.redhat.com/show_bug.cgi?id=1676824[(BZ#1676824)]
 +
-** This will be fixed with the introduction of operators in CNV 1.4.
+[NOTE]
+====
+The command `$ ls -sh <image_file>` reports the actual size. For uncompressed raw
+images, the command `$ ls -lh <image_file>` reports the virtual size. For QCOW2
+images, use `$ qemu-img info <image_file>` to get both values.
+====
+
+* If you create a new DataVolume while a PVC already exists with the same name,
+the DataVolume enters an unrecoverable error state. If this DataVolume is
+associated with a virtual machine, or if it was created with the
+`dataVolumeTemplates` section of a VM configuration file, then the VM will fail
+to start. In these cases, the underlying DataVolume error will not be propagated
+to the VM. https://bugzilla.redhat.com/show_bug.cgi?id=1669163[(BZ#1669163)]
+
+* If a CDI import into a PVC fails, a request to delete the PVC might not work
+immediately. Instead, the importer Pod gets stuck in a `CrashLoopBackOff` state,
```

Comment:
s/importer Pod/importer pod/

Reply:
Ahh, I forgot that it was "pod" in 3.11. I had recently referred to the 4.0 glossary where it's listed as "Pod." Thanks, @aburdenthehand :)